EIPS/eip-1815.md

```diff
@@ -0,0 +1,65 @@
+Many dApps that use blind auctions have appeared in the last years, each one reinventing the wheel and implementing blind auctions in ways that are conceptually similar but with special quirks and different interfaces, thus making it impossible for a system to interact with all of these without accommodating all these differences. This ERC aims to solve that by standarizing an interface for blind auctions contracts, which will enable the creation of systems that can interact with any blind auctions without requiring specialized code.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+```
+interface BlindAuction {
+    enum Mode { Open, Auction, Owned, Forbidden, Reveal, NotYetAvailable }
+
+    event AuctionStarted(bytes32 indexed hash, uint registrationDate);
+    event NewBid(bytes32 indexed hash, address indexed bidder, uint deposit);
+    event BidRevealed(bytes32 indexed hash, address indexed owner, uint value, uint8 status);
+    event HashRegistered(bytes32 indexed hash, address indexed owner, uint value, uint registrationDate);
+
+    function startAuction(bytes32 _hash) external;
+    function startAuctions(bytes32[] calldata _hashes) external;
+    function newBid(bytes32 sealedBid) external payable;
+    function startAuctionsAndBid(bytes32[] calldata hashes, bytes32 sealedBid) external payable;
+    function unsealBid(bytes32 _hash, uint _value, bytes32 _salt) external;
+    function cancelBid(address bidder, bytes32 seal) external;
+    function finalizeAuction(bytes32 _hash) external;
+    function entries(bytes32 _hash) external view returns (Mode, address, uint, uint, uint);
+}
+```
+
+## Rationale
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+The interface proposed has been chosen because it is the one used by the Ethereum Name System, which is the most popular project among the ones that use blind auctions, therefore being the one that has the most systems that integrate with it. By picking this interface we aim at minimizing the total amount of work that will be needed to migrate existing systems to the interface proposed, while maximizing the number of systems that will already be compatible with a contract implementing the interface described.
```

Comment:
If this is an exact copy of what ENS is using, maybe it would be courteous to include the ENS authors in the author field of the ERC?

Reply:
I've previously contacted some of them and they didn't seem interested in being authors but I'll try to contact the others.